modules/helm-understanding-helm.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="understanding-helm_{context}"]
+= Understanding Helm
+
+Helm is a CLI tool the simplifies deployment of applications and services to {prod-title} clusters. Helm uses a packaging format called charts - a collection of files that describe {prod-title} resources. 
```

Comment:
My apologies @boczkowska, in my comment I didn't properly say what `{prod-title}` was supposed to change to. It should be `{product-title}`. Again, when you look at the built version, you should be able to see that these attributes weren't replaced properly.